src/core/attribution/markovChain.js

```diff
@@ -7,6 +7,17 @@
  */
 export type Distribution = Float64Array;
 
+export type StationaryDistributionResult = {|
+  // The final distribution after attempting to find the stationary distribution
```

Comment:
Please capitalize “Markov chain” and end these sentences with periods.

Reply:
Done.